core/src/OpenMPTarget/Kokkos_OpenMPTarget.hpp

```diff
@@ -148,7 +150,9 @@ struct DeviceTypeTraits<::Kokkos::Experimental::OpenMPTarget> {
 #include <OpenMPTarget/Kokkos_OpenMPTarget_Parallel.hpp>
 #include <OpenMPTarget/Kokkos_OpenMPTarget_ParallelFor_MDRange.hpp>
 #include <OpenMPTarget/Kokkos_OpenMPTarget_ParallelReduce_MDRange.hpp>
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
 #include <OpenMPTarget/Kokkos_OpenMPTarget_Task.hpp>
```

Comment:
Does it make sense to keep this files around? You removed `Kokkos_OpenMPTarget_Task.cpp`.

Reply:
That's fair, see c9f1e51c4.